### ISSUE TITLE
TypeError on request /home/ when not logged in.

### ISSUE THREAD

**Issue by brenns10** [jpcjr/corvid#11]
^

**Comment by brenns10** [comment]
```
Environment:


Request Method: GET
Request URL: http://localhost:8000/home/

Django Version: 1.8.5
Python Version: 3.5.0
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'main')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.security.SecurityMiddleware')


Traceback:
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/core/handlers/base.py" in get_response
  132.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/views/generic/base.py" in view
  71.             return self.dispatch(request, *args, **kwargs)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/views/generic/base.py" in dispatch
  89.         return handler(request, *args, **kwargs)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/views/generic/list.py" in get
  159.         self.object_list = self.get_queryset()
File "/home/stephen/repos/corvid/corvid/main/views/user.py" in get_queryset
  13.         return Project.objects.filter(owner=self.request.user)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/manager.py" in manager_method
  127.                 return getattr(self.get_queryset(), name)(*args, **kwargs)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/query.py" in filter
  679.         return self._filter_or_exclude(False, *args, **kwargs)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/query.py" in _filter_or_exclude
  697.             clone.query.add_q(Q(*args, **kwargs))
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/sql/query.py" in add_q
  1310.         clause, require_inner = self._add_q(where_part, self.used_aliases)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/sql/query.py" in _add_q
  1338.                     allow_joins=allow_joins, split_subq=split_subq,
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/sql/query.py" in build_filter
  1200.                                                     lookups, value)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/fields/related.py" in get_lookup_constraint
  1745.                     lookup_class(target.get_col(alias, source), val), AND)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/lookups.py" in __init__
  101.         self.rhs = self.get_prep_lookup()
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/lookups.py" in get_prep_lookup
  139.         return self.lhs.output_field.get_prep_lookup(self.lookup_name, self.rhs)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/fields/__init__.py" in get_prep_lookup
  727.             return self.get_prep_value(value)
File "/home/stephen/repos/corvid/venv/lib/python3.5/site-packages/django/db/models/fields/__init__.py" in get_prep_value
  985.         return int(value)

Exception Type: TypeError at /home/
Exception Value: int() argument must be a string, a bytes-like object or a number, not 'SimpleLazyObject'
```

**Comment by jpcjr** [comment]
Resolved in #13
